lib/sqlite_ecto/error.ex

```diff
@@ -8,5 +8,15 @@ defmodule Sqlite.Ecto.Error do
   def exception({type, msg}), do: %__MODULE__{sqlite: {type, to_string(msg)}}
   def exception(msg), do: %__MODULE__{sqlite: msg}
 
+  require Logger
+
+  def to_constraints(%__MODULE__{sqlite: {:constraint, "UNIQUE constraint failed: " <> _}}) do
```

Comment:
Keep in mind this will be called every time there is such exceptions, even if not using unique_constraint/3. So if the user is rescueing the exception, they will see the warning. That's why I thought about extending the exception message in the exception function above. But at the end of the day it is your call. :)